vendor/vendor.json

```diff
@@ -34,6 +34,12 @@
 			"revision": "093482f3f8ce946c05bcba64badd2c82369e084d",
 			"revisionTime": "2018-02-27T14:14:24Z"
 		},
+		{
+			"checksumSHA1": "aal26fgtxj7sXIkgKAZwOZuNLg0=",
+			"path": "github.com/go-zookeeper/zk",
+			"revision": "54f48128ea8ce6e8431de54415a1f965757ddb65",
```

Comment:
just for coriosity i checked that this is from the last master commit, looks fine